src/components/NavBar.vue

```diff
@@ -0,0 +1,371 @@
+<template>
+  <nav class="bg-white shadow">
+    <div class="max-w-7xl mx-auto px-2 sm:px-4 lg:px-8">
+      <div class="flex justify-between h-16">
+        <div class="flex px-2 lg:px-0">
+          <div class="flex-shrink-0 flex items-center">
+            <img
+              class="block lg:hidden h-8 w-auto"
+              :src="require(`../assets/images/${imgURL}`)"
```

Comment:
You definitely don't want to be importing the image this way. This is basically embedding the asset in the built nuxt app, making it much heavier than necessary. Images should always be loaded client-side, on demand as needed. This should be a path to the static asset. You're trying to load this in as a dynamic image, but that's definitely overkill for this.

Reply:
changed paths to static assets, removed dynamic.